tests/acceptance/features/bootstrap/WebUIBruteForceProtectionContext.php

```diff
@@ -0,0 +1,128 @@
+<?php
+/**
+ * ownCloud
+ *
+ * @author Artur Neumann <info@jankaritech.com>
+ * @copyright Copyright (c) 2018 Artur Neumann info@jankaritech.com
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU AFFERO GENERAL PUBLIC LICENSE
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU AFFERO GENERAL PUBLIC LICENSE for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public
+ * License along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+use Behat\Behat\Context\Context;
+use Behat\Behat\Hook\Scope\BeforeScenarioScope;
+use Behat\Gherkin\Node\TableNode;
+use Behat\MinkExtension\Context\RawMinkContext;
+use Page\OwncloudPage;
+use Page\BruteForceProtectionPage;
+use Page\LoginPage;
+
+require_once 'bootstrap.php';
+
+/**
+ * Context for brute force protection specific webUI steps
+ */
+class WebUIBruteForceProtectionContext extends RawMinkContext implements Context {
+
+	/**
+	 * @var FeatureContext
+	 */
+	private $featureContext;
+	
+	/**
+	 *
+	 * @var WebUIGeneralContext
+	 */
+	private $webUIGeneralContext;
+
+	/**
+	 *
+	 * @var OwncloudPage
+	 */
+	private $owncloudPage;
+
+	/**
+	 *
+	 * @var BruteForceProtectionPage
+	 */
+	private $bruteForceProtectionPage;
+
+	/**
+	 *
+	 * @var LoginPage
+	 */
+	private $loginPage;
+	/**
+	 * WebUIBruteForceProtectionContext constructor.
+	 *
+	 * @param OwncloudPage $owncloudPage
+	 * @param BruteForceProtectionPage $loginPage
```

Comment:
@individual-it this param is mixed up - should be 2 lines

Reply:
PR #20
php-cs-fixer did not shout at me, even the count of `@param` was not correct :-1: